extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -63,10 +63,17 @@ class AmpTwitter extends AMP.BaseElement {
     this.applyFillContent(iframe);
     listenFor(iframe, 'embed-size', data => {
       // We only get the message if and when there is a tweet to display,
-      // so hide the placeholder.
+      // so hide the placeholder
       this.togglePlaceholder(false);
       this./*OK*/changeHeight(data['height']);
     }, /* opt_is3P */true);
+    listenFor(iframe, 'no-content', () => {
+      if (this.getFallback()) {
```

Comment:
Nit: We could check this outside the listener, to avoid creating the listener when there's no fallback:

```js
if (this.getFallback()) {
  listenFor(iframe, 'no-content', () => {});
}